### DISCUSSION TITLE
最新版本终于来了. 7.8.5.为什么每一次升级都轮不到WIN7版本？

### DISCUSSION THREAD

**Discussion by ycjdixon** [getlantern/lantern-forum-cn#453] [General]
WIN7系统还要等多久才能正常使用？已经非正常好几个月。

**Comment by MaxMKMax** [comment]
@ycjdixon 您好, 那天我又和技术团队开会了. 我们已经把window 7 的需求升级到紧急需要了. 他们会关注了! 
不好意思让您久等了, 应为之前没想到 Window 7 这么多人有需求.

**Reply by MaxMKMax** [reply]
@ycjdixon 技术团队已正式关注 Window 7 的用户了. 谢谢您耐心的等待. 应该不久后 Window 7 就能用了.

**Reply by ycjdixon** [reply]
从去年年底等到现在，已经很久了。中间就正常了没几天。每天在浪费钱。还要找其他免费的VPN接上。

**Reply by 396738631** [reply]
win7 很多老用户啊，这个请及时提上日程，最新下载的根本就还是不能用的7.8.2版本的
